src/components/Form/Form.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react';
+import usersFromServer from '../../api/users';
+import todosFromServer from '../../api/todos';
+import { Todo } from '../../types/types';
+
+type Props = {
+  addNewTodo: (todo: Todo) => void;
+};
+
+export const Form = ({ addNewTodo }: Props) => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [chooseUser, setChooseUser] = useState(0);
+  const [hasUserError, setHasUserError] = useState(false);
+
+  const todos: Todo[] = todosFromServer.map(todo => ({
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId) || null,
+  }));
+
+  function createGetNextId() {
```

Comment:
It is important to follow the same way of creating functions everywhere
```suggestion
  const createGetNextId = () => {
```